### DISCUSSION TITLE
基于umi的模板antd-admin做的项目，打开chrome开发者工具，刷新页面加载很慢的原因？

### DISCUSSION THREAD

**Discussion by mayunyi** [umijs/umi#11816] [Q&A]
1. 使用antd-admin模板开发大型项目，使用chrome，正常情况下不打开开发者工具页面加载速度正常
2. 打开chrome的开发者工具，刷新页面加载很慢。
3. 使用Safari浏览器打开开发者工具，和不打开开发者工具加载速度一样
4. 在windos和mac下测试chrome存在一样的问题
chrome版本：118.0.5993.117
mac版本： big sur 11.7.4
windows: 11

需求：解决在开发的时候，打开开发者工具，加载速度和不打开开发者工具的速度一样

**Comment by mayunyi** [comment]
![image](https://github.com/umijs/umi/assets/26814094/6b6126f4-a41b-4fbf-9fe6-0fdbf9f039c6)

标记1的请求的是标记3的数据，这些加载完期间存在很长的空白区

**Reply by fz6m** [reply]
3 的脚本加载时间都是 100ms 左右，如果认为是前置请求阻塞了加载，自己排查下接口请求速度的问题吧，另外可以看下有没有加载其他网络的资源阻塞了后续脚本加载。

如果是 antd-admin 的资源或者接口影响到的，可以去对应仓库反馈。

如还有问题，需要给一个最小复现。

**Comment by mayunyi** [comment]
![image](https://github.com/umijs/umi/assets/26814094/6ec5d5d4-6c47-48fa-ae46-87a3f801a157)

![image](https://github.com/umijs/umi/assets/26814094/99d9d8c0-aded-4291-a903-4d051151b2b5)
![image](https://github.com/umijs/umi/assets/26814094/fee17da3-4b58-47b2-85df-7180ddb9faa3)
![image](https://github.com/umijs/umi/assets/26814094/a1732a32-e135-43fe-9420-7522c02d61c6)
![image](https://github.com/umijs/umi/assets/26814094/ad1c66d7-2db0-4116-8033-576769856149)


这是前几个加载的情况，不知道能不能看出什么问题

**Reply by isbox** [reply]
你看看配置你有没有mfsu，关掉它试试。感觉是chrome的bug，mfsu拆了很多小文件，刷新进入时大量的请求发送会让浏览器卡住（devtools模式下），试了下safari浏览器就没这问题
